packages/react-native-editor/react-native.config.js

```diff
@@ -0,0 +1,12 @@
+const path = require( 'path' );
+
+module.exports = {
```

Comment:
This is needed to solve an issue after the update, where the code that auto-links dependencies was picking up the wrong path `gutenberg/node_modules/@wordpress/`. So when running `pod install` it would change the path as seen in previous commits, for example [this one](https://github.com/WordPress/gutenberg/pull/44889/commits/2440f5a0a73ea2432a8df6d9660537482bb6f639#diff-ce3a290cf4efc62a6f6097db3b4f49ba4799ca7c66f599bc16d94ed93586028cL384).